package.json

```diff
@@ -24,7 +24,7 @@
     "acorn-globals": "^4.1.0",
     "array-equal": "^1.0.0",
     "cssom": ">= 0.3.2 < 0.4.0",
-    "cssstyle": ">= 0.2.37 < 0.3.0",
+    "cssstyle": "^0.3.1",
```

Comment:
`^` doesn't really work with `0.` versions. `>= 0.3.1 < 0.4.0` should work just fine.

Reply:
We should probably stick to exact versions actually until we have a 1.0.0 that's guaranteed to not make breaking changes...

Reply:
@domenic - That is what I had but @TimothyGu requested a change. I'm happy to pin it at 0.3.1 again.

There won't be any breaking API changes until I have time to do a re-write. And even then I think the API will probably be the same, I am just going to generate the properties in more scalable way.

Reply:
With @jsakas's guarantee, I think having a version range is fine.